### DISCUSSION TITLE
Consider moving away from Carthage for development purposes

### DISCUSSION THREAD

**Discussion by BasThomas** [Moya/Moya#2184] [General]
Currently, we use Carthage to set up our project's dependencies, which are required for the project to build and test.

I think we may have decided on doing so in the past as a way of making sure our Carthage integration works as expected; it's presumably the least used dependency manager between CocoaPods, Carthage and now Swift Package Manager.

However, it seems to currently be broken [locally](https://github.com/Moya/Moya/pull/2183), or at least require changes to make things work. And on top of that, it takes a very, very long time (easily 30m+) to set up your environment, which isn't the most welcoming to people trying to help out contributing to the project.

Thoughts? I'd love to move to something like SwiftPM for development purposes.

**Comment by AndrewSB** [comment]
+1 for SPM
